test/parse.jl

```diff
@@ -315,3 +315,11 @@ end
     @test Meta.parse("±x") == Expr(:call, :±, :x)
     @test Meta.parse("∓x") == Expr(:call, :∓, :x)
 end
+
+@testset "test .<: and .>:" begin
+    tmp = [Int, Float64, String, Bool] .<: Union{Int, String}
+    @test tmp == Bool[1, 0, 1, 0]
+
+    tmp = [Int, Float64, String, Bool] .>: [Int, Float64, String, Bool]
+    @test tmp == Bool[1, 1, 1, 1]
```

Comment:
Maybe also test `tmp = @. [Int, Float64, String, Bool] <: Union{Int, String}

Reply:
Added a test in a3e3ec4